doc/source/_static/css/getting_started.css

```diff
@@ -236,7 +236,7 @@ ul.task-bullet > li > p:first-child {
 
 .tutorial-card .card-header {
   cursor: pointer;
-  background-color: white;
+  background-color: transparent;
```

Comment:
Is this related to #48527 ?

Reply:
yes it is, sorry for not mentioning this (first timer here)